cuppa-surefire/src/main/java/org/forgerock/cuppa/maven/surefire/CuppaSurefireProvider.java

```diff
@@ -81,21 +99,10 @@ public Iterator getSuites() {
         return scanClasspathForTests().iterator();
     }
 
-    private TestsToRun scanClasspathForTests() {
-        return providerParameters.getScanResult()
+    private List<Class<?>> scanClasspathForTests() {
+        return Arrays.asList(providerParameters.getScanResult()
                 .applyFilter(clazz -> Arrays.stream(clazz.getAnnotations())
-                        .anyMatch(annotation -> Test.class.equals(annotation.annotationType())),
-                        providerParameters.getTestClassLoader());
-    }
-
-    private void instantiateTestClasses(TestsToRun testClasses) {
-        Arrays.stream(testClasses.getLocatedClasses()).forEach(testClass -> {
-            try {
-                CuppaTestProvider.setTestClass(testClass);
-                testClass.newInstance();
-            } catch (InstantiationException | IllegalAccessException e) {
-                throw new IllegalStateException("Must be able to instantiate test classes", e);
-            }
-        });
+                                .anyMatch(annotation -> Test.class.equals(annotation.annotationType())),
```

Comment:
Much as I like the lambdas, there's a way of doing this straight from the `Class` I think - `clazz -> clazz.getAnnotation(Test.class) != null`